app/views/landing_page/blocks/_statistics.html.erb

```diff
@@ -20,5 +20,4 @@
   point_size: block.data["point_size"],
   h_axis_format: block.data["x_axis_format"],
   v_axis_format: block.data["y_axis_format"],
-  hide_legend: block.data["hide_legend"]
```

Comment:
Hmm suspect this commit was intended for a different PR?!

Reply:
+1 This is not related to web banners.

Reply:
Lol, I don't know how this got in this PR! Anyway, removed now.